src/tests/class.test.js

```diff
@@ -46,3 +49,53 @@ describe('GET /api/classes', () => {
     expect(response.error).toBeFalsy();
   });
 });
+
+describe('GETBYID /api/classes/:id', () => {
+  test('Should return status 200 and false error.', async () => {
+    const response = await request(app).get(`/api/classes/${classId}`).send();
+    expect(response.status).toBe(200);
+    expect(response.body.error).toBeFalsy();
+  });
+  test('Should return an object containing a message, a type of error, and data.', async () => {
+    const response = await request(app).get(`/api/classes/${classId}`).send();
+    expect(response.body).toBeInstanceOf(Object);
+    expect(response.body).toHaveProperty('data');
+    expect(response.body).toHaveProperty('message');
+    expect(response.body).toHaveProperty('error');
+  });
+  test('Should return data as an object.', async () => {
+    const response = await request(app).get(`/api/classes/${classId}`).send();
+    expect(typeof response.body.data).toBe('object');
+    expect(response.body.error).toBeFalsy();
+  });
+  // eslint-disable-next-line quotes
+  test(`Should return a message indicating "Class ${classId} obtained".`, async () => {
+    const response = await request(app).get(`/api/classes/${classId}`).send();
+    expect(response.body.message).toBe(`Class ${classId} obtained.`);
+    expect(response.body.error).toBeFalsy();
+  });
+  test('Should return all properties from the class.', async () => {
+    const response = await request(app).get(`/api/classes/${classId}`).send();
+    const classItem = response.body.data;
+    expect(classItem).toHaveProperty('day');
```

Comment:
here you could do this:

`const expectedProperties = ['day', 'hour', 'trainer', 'activity', 'slots'];
    expectedProperties.forEach((property) => {
      expect(classItem).toHaveProperty(property);
    });`